recipes/pybind11_json/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class Pybind11JsonConan(ConanFile):
+    name = "pybind11_json"
+    homepage = "https://github.com/pybind/pybind11_json"
+    description = "An nlohmann_json to pybind11 bridge"
+    topics = (
+        "conan",
+        "header-only",
+        "json",
+        "nlohmann_json",
+        "pybind11",
+        "pybind11_json",
+        "python",
+        "python-binding",
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    no_copy_source = True
+    license = "BSD-3-Clause"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def package_id(self):
+        self.info.header_only()
+
+    def requirements(self):
+        self.requires("nlohmann_json/3.9.1")
+        self.requires("pybind11/2.6.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def package(self):
+        self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
+        self.copy(
+            "*", dst="include", src=os.path.join(self._source_subfolder, "include")
+        )
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "pybind11_json"
+        self.cpp_info.names["cmake_find_package_multi"] = "pybind11_json"
```

Comment:
```suggestion
```
Why not leave it on default, the packagename is the same as the generator name?

Reply:
It's a message to maintainers that somebody had a look and verified that the upstream project creates these files.
It's no certainty that conan will generate files for recipes that don't explicitly set these properties.

Reply:
What? This just sets the name for the generateors **in case** someone uses that specific gen. What is the message here, that `CMakeDeps` output was not yet reviewed? Why should this matter at all, conan is exactly trying to decouple whatever the project itself does with its generators.

Or maybe I completely miss your point somehow.

Reply:
Indeed, this is set in case someone uses those generators.
By explicitly setting the name for the `cmake_find_package` generators, a recipe author tells the cci maintainers that he manually checked what cmake scripts the build script generates.
In pybind11_json's case, [cmake installs these files](https://github.com/pybind/pybind11_json/blob/0fbbe3bbb27bd07a5ec7d71cbb1f17eaf4d37702/CMakeLists.txt#L130-L132).

I asked you to remove setting the `pkg_config` name, because [no `.pc` file is installed by the cmake script](https://github.com/pybind/pybind11_json/search?q=.pc).

No recipes at cci use/test CMakeDeps or set properties.
I would even dare to say that `CMakeDeps` is not supported atm.

Looking at the documentation of [`CMakeDeps`](https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmakedeps.html), it looks like it only generates `XXX-config.cmake` modules.
Some/few/a lot of cmake scripts require dependencies to be provided by `FindXXX.cmake` modules.
Just to say that CMakeDeps is not a complete solution.